src/d3a/models/strategy/area_agents/two_sided_revenue_fee.py

```diff
@@ -0,0 +1,96 @@
+
+
+def update_incoming_bid_with_fee(source_bid, original_bid, tax_percentage=None):
```

Comment:
`tax_percentage` is not used here. Is it kind of a placeholder?

Reply:
That was what I was intending to do. For now this is not useful, but I wanted to have a future-proof interface that would allow this module to be replaced with a custom fee calculation model. In this case, the tax_percentage would be needed. 
I did not finish implementing this though, since I was rushing to wrap up this story. But if you also agree, I think it would be nice to have a proper interface (using this https://docs.python.org/3/library/abc.html for instance) that would oblige the one who replaces the market fee to follow the same interface as here. Let me know if you agree, so that I can add it in this PR.